docs/assets/scss/_styles_project.scss

```diff
@@ -61,3 +61,15 @@
     padding-top: 4rem;
```

Comment:
Yes, the css change is needed for this page's tabs to be rendered correctly. It will have no impact on existing docs as the `.tab-content` class was not used anywhere before